src/core/search/searchdatatransformer.js

```diff
@@ -30,42 +30,46 @@ export default class SearchDataTransformer {
     };
   }
 
-  static transformVertical (data, formatters, verticalKey) {
```

Comment:
Was the `formatters` logic moved elsewhere? While we don't want any concept of `formatters` in the Core library, this will still need to work in the SDK.

Reply:
That's on me! I suspect Alexis was following what I did in the universal adapter which was to leave them out. I planned on getting them in a future PR, but that got put on the backburner because of the theme release. At this point, we can get the formatters in with this PR (and update the universal search as well), or we can do it in separate PR

Reply:
Exactly, since the "ResultsFactory.from" was removed in the feature branch, the formatters arg was unused. I can add it back here, but we'll likely need a new PR to actually apply the formatters to the results

Reply:
Added it back with a "TODO" in the Section's constructor!